tox.ini

```diff
@@ -13,9 +13,10 @@ deps =
   sphinx210: sphinx==2.1.0
   sphinx220: sphinx==2.2.0
   sphinx231: sphinx==2.3.1
+  sphinx244: sphinx==2.4.4
```

Comment:
Maybe do `sphinx<2.5;>2.4` or w/e that syntax is?

Reply:
Yeah. The syntax is `sphinx~=1.8.0`. It will install 1.8.x